feat: add outputDirPath arg to CLI

jlowhy opened the pull request:
### What
- Provide an arg to set a custom output path for remix-routes.d.ts

### Why
- For our remix project, we have specific folders where builds and types end up. As such, it would be helpful to be able to specify an output path other than `node_modules`. This should also be relevant to others who are using remix-routes.

yesmeck commented:
Just curious, if you put `remix-routes.d.ts` somewhere other than in `node_modules`, how do you tell TypeScript to find it?

jlowhy commented:
Hi, we set this up in our `tsconfig.json` file by specifying the `paths` and `include` options.

So our tsconfig looks something like this:
```
{
  "compilerOptions": {
  ...
    "moduleResolution": "Node",
    "paths": {
      "~/*": ["./app/*", "./build/*"]
    },
  "include": ["**/*.ts", "**/*.tsx"]
}
```

In our case, we would place the generated `remix-routes.d.ts` file in our `build` directory, which is specifically where we put our build artifacts. For example, we use Prisma and we direct the artifacts from `prisma generate` into the `build` directory.

This helps us have a clear place to manage build artifacts. We prefer to keep this separate from the `node_modules` folder.

yesmeck commented:
Could you please update the readme as well?

jlowhy commented:
Hey, I updated the README. Let me know if it's clear enough.
I also made a slight change to the default arg from `node_modules` to `./node_modules` instead and updated the test accordingly to make sure that relative pathing will work since we should expect inputs like `./<dirName>`.

Thanks!

yesmeck commented:
👍 Thanks, released in 1.3.0.
